### PR TITLE
Workbench Improvements

### PR DESCRIPTION
**Pull request by crynobone**

### CODE REVIEW

File: tests/TestCase.php [hunk 1]
```diff
@@ -3,30 +3,23 @@
...
         TurboStream::fake();
     }
 
-    protected function getPackageProviders($app)
-    {
-        return [
-            TurboServiceProvider::class,
-            WorkbenchAppServiceProvider::class,
-        ];
-    }
```
Comment (crynobone) [inline tests/TestCase.php:27]: Both `getPackageProviders` and `loadMigrationsFrom` will rely on `testbench.yaml` values when we use `Orchestra\Testbench\Concerns\WithWorkbench`

File: tests/TestCase.php [hunk 2]
```diff
@@ -3,38 +3,25 @@
...
-    public function getEnvironmentSetUp($app)
-    {
-        $app['config']->set('database.default', 'sqlite');
-
-        $app['config']->set('database.connections.sqlite', [
-            'driver' => 'sqlite',
-            'database' => ':memory:',
-            'prefix' => '',
-        ]);
+        $app['config']->set('database.default', 'testing');
```
Comment (crynobone) [inline tests/TestCase.php:24]: `testing` database connections is always available using `:memory:` sqlite.

File: tests/TestCase.php (hunk 1 above)
Comment (tonysm) [reply tests/TestCase.php:27]: Oh, nice!

**Comment by tonysm** [comment]
Thank you so much!

**Pull request merged**